default.yml

```diff
@@ -68,3 +40,59 @@ Style/PercentLiteralDelimiters:
 
 Style/RegexpLiteral:
   Enabled: false
+
+#################### Naming ################################
+
+Naming/FileName:
+  Enabled: false
+
+#################### Metrics ###############################
+
+Metrics/AbcSize:
+  Max: 22.5
+
+Metrics/BlockLength:
+  Exclude:
```

Comment:
Missing Max attribute here.

Reply:
The max is already inherited from the default config, so I thought I'd leave it off. I can add it if you want though.

Reply:
Yes, I was think in updating Max to 38 (50% more)

Reply:
It's weird that by default, block length can be 25, but method length can only be 10.

Reply:
Maybe it's for scripts? It's confusing to me as well. That's kind of why I figured I'd just leave it as is.